configuration/analyzer_config.json

```diff
@@ -209,7 +213,10 @@
     "decription": "static generic document analysis (Oletools)",
     "python_module": "doc_info.DocInfo",
     "soft_time_limit": 30,
-    "queue": "local"
+    "queue": "local",
+    "additional_config_params": {
```

Comment:
I don't think this is required here. The `additional_passwords_to_check` is a feature of the `Doc_Info_Experimental` only.